lib/volk_malloc.c

```diff
@@ -50,6 +50,12 @@
 
 void* volk_malloc(size_t size, size_t alignment)
 {
+    // The ASAN address sanitizer requires the allocation size to be a multiple of
+    // the the alignment. Add bytes to size so that it is a multiple of alignment
+    // if it's not.
+    if (size % alignment) {
```

Comment:
Please add the following code (correctly formatted) just before this section. We need this before the original code from this PR to make sure the math in the PR is valid. We didn't need it before because we passed any math issues into other functions.
```
if ((size == 0) || (alignment == 0)) {
    fprintf(stderr, "VOLK: Error allocating memory: either size or alignment is 0");
    return NULL;
}
```